<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>544</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MySafe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="3,3,1,7,1">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout" columnminimumwidth="0,0,0,0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="websiteNameLineEdit">
          <property name="placeholderText">
           <string>Website</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLineEdit" name="emailLineEdit">
          <property name="placeholderText">
           <string>E-mail</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="passwordLineEdit">
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="websiteLinkLineEdit">
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Link</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="lastNameLineEdit">
          <property name="placeholderText">
           <string>Last Name</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="mainDeviceLineEdit">
          <property name="placeholderText">
           <string>Main Device</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="securityQuestionLineEdit">
          <property name="placeholderText">
           <string>Security Question 1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLineEdit" name="answerLineEdit">
          <property name="placeholderText">
           <string>Answer 1</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QLineEdit" name="purposeOfUseLineEdit">
          <property name="placeholderText">
           <string>Purpose of Use</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="firstNameLineEdit">
          <property name="placeholderText">
           <string>First Name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLineEdit" name="linkedWebsiteLineEdit">
          <property name="placeholderText">
           <string>Linked Website</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLineEdit" name="phoneNumerLineEdit">
          <property name="placeholderText">
           <string>Phone Number</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLineEdit" name="usernameLineEdit">
          <property name="placeholderText">
           <string>UserName</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLineEdit" name="secondEmailLineEdit">
          <property name="placeholderText">
           <string>Second E-mail</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDateEdit" name="birthDateEdit">
          <property name="dateTime">
           <datetime>
            <hour>0</hour>
            <minute>0</minute>
            <second>0</second>
            <year>1993</year>
            <month>1</month>
            <day>1</day>
           </datetime>
          </property>
          <property name="maximumDate">
           <date>
            <year>2100</year>
            <month>12</month>
            <day>31</day>
           </date>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="notePlainTextEdit">
        <property name="placeholderText">
         <string>Notes</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="specialDigitsCheckBox">
          <property name="text">
           <string>Special Digitis</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="numbersCheckBox">
          <property name="text">
           <string>Numbers</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="upperCaseCheckBox">
          <property name="text">
           <string>Upper Case</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="lowerCaseCheckBox">
          <property name="text">
           <string>Lower Case</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="lengthPasswordSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="generatePasswordPushButton">
          <property name="text">
           <string>Generate Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addPushButton">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="allDataTab">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <attribute name="title">
          <string>All Data</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QTableWidget" name="allDataTableWidget">
            <property name="gridStyle">
             <enum>Qt::SolidLine</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>90</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>id</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Website</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Link</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Linked Website</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>E-mail</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Second Email</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>UserName</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>First Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Last Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Birthdate</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Phone Number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Password</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Security Question</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Answer</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Main Device</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Purpose Of Use</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QTableWidget" name="tableWidget_2">
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>90</number>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>#</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Website</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Link</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Linked Website</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>E-mail</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Second Email</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>UserName</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>First Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Last Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Birthdate</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Phone Number</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Security Question</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Answer</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Main Device</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Purpose Of Use</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="copyPushButton">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deletePushButton">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="updatePushButton">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="visualizeDataPushButton">
          <property name="text">
           <string>Visualize Data</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionHistory"/>
    <addaction name="actionChange_Password"/>
    <addaction name="separator"/>
    <addaction name="actionLog_Out"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionHowToUse"/>
    <addaction name="actionAbout_this_Software"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHistory">
   <property name="text">
    <string>History</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="text">
    <string>Change Password</string>
   </property>
  </action>
  <action name="actionLog_Out">
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionImprot">
   <property name="text">
    <string>Improt</string>
   </property>
  </action>
  <action name="actionDelete_All">
   <property name="text">
    <string>Delete All Passwords On This Account</string>
   </property>
  </action>
  <action name="actionDelete_Everything">
   <property name="text">
    <string>Delete Everything</string>
   </property>
  </action>
  <action name="actionHowToUse">
   <property name="text">
    <string>How to use?</string>
   </property>
  </action>
  <action name="actionAbout_this_Software">
   <property name="text">
    <string>About this Software</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
